Scripts/Modules/MultipleRayInputModule/MultipleRayInputModule.cs

```diff
@@ -211,9 +210,9 @@ public LayerMask layerMask
         // Local method use only -- created here to reduce garbage collection
         RayEventData m_TempRayEvent;
 
-        protected override void Awake()
+        protected override void OnEnable()
         {
-            base.Awake();
+            base.OnEnable();

```

Comment:
I believe this is with how initialization had to change for the script runtime because all of the default script references now already exist on a prefab to be instantiated from and enabled rather than how they are created from the editor.

Reply:
Aha. Ok I will test going back to Awake to see if/what breaks. We may be able to still use awake if we set the default script references script to have a super-low execution order. This was how we allowed for using FI-dependent code in MARS--the script which injects functionality on everything forces itself into a very low execution order.

Reply:
I tried reverting all of the modules where Awake changed into OnEnable and I found that Awake was still called in Play Mode and Edit Mode. Oddly, I found that EditorVR.Awake doesn't get called in Play Mode, which it seems is why you changed Awake to Initialize, and call it in the beginning of Start. I'll look into this later, but for now, we can safely revert the modules and leave EditorVR as it is.